config.toml

```diff
@@ -407,6 +407,12 @@ defaultContentLanguage = "en"
     identifier = "chart-repositories-frequently-asked-questions"
     weight = 208
     parent = "developing_charts"
+  [[menu.main]]
+    name = "Registries"
```

Comment:
Can we give a more descriptive title? Perhaps "OCI Registry Guide" similar to the Chart Repository Guide?

Reply:
done!